web/concrete/src/File/Image/BasicThumbnailer.php

```diff
@@ -15,6 +14,39 @@ class BasicThumbnailer
     protected $jpegCompression;
 
     /**
+     * @var StorageLocation
+     */
+    private $storageLocation;
+
+    /**
+     * BasicThumbnailer constructor.
+     * Set the storage location to the the default, if it exists.
+     */
+    public function __construct()
+    {
+        $sl = StorageLocation::getDefault();
+        if(is_object($sl)) {
+            $this->setStorageLocation($sl);
+        }
```

Comment:
:+1: 

I also think that `$storageLocation` should be `protected` for this matter. If I want to override the `getStorageLocation()` method, I probably want to be able to check if that's set without calling the parent implementation, which would in turn set it.